code/script.js

```diff
@@ -30,18 +37,156 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+//Question / answer flow
+
+const handleInput = (event) => {     
+  event.preventDefault()
+  console.log(currentQuestion)
+  if(currentQuestion === 1){
+    handleNameQuestion()
+  } else if(currentQuestion === 2){
+    moodOption()
+  } else if(currentQuestion === 3){
+    userMood()
+  } else if(currentQuestion === 4){
+    gif()
+  } else if(currentQuestion === 5){
+    moodGif()
+  } else if (currentQuestion === 6){
+    byeRestart
+  }
+}
+
+
+// Questions / answers starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
+  showMessage(`Hello there! What's your name?`, 'bot')
   // Just to check it out, change 'bot' to 'user' here 👆
 }
 
+const handleNameQuestion = () => {  
+  currentQuestion = 1
+  const name = input.value
+    console.log(name)
+    showMessage(name, 'user')
+    input.value = ''
+    showMessage (`Hello, ${name}! Lets talk.`, 'bot')
+    setTimeout(moodOption, 1000)
```

Comment:
setTimout is in every function, this is a function that starts the function below. It is set to appear with 1sec delay, but on the website the message is not delayed with 1sec. This indicates that this setTimout is not in play and the next message starts with the nice function you declared above (handeInput). So you just need one of the two (setTimeout or handleInput).